cg/constants/gene_panel.py

```diff
@@ -44,3 +46,22 @@ class GenePanelMasterList(StrEnum):
     def get_panel_names(cls, panels=None) -> list[str]:
         """Return requested panel names from the Master list, or all panels if none are specified."""
         return list(panels) if panels else list(cls)
+
+    @staticmethod
+    def collaborators() -> set[str]:
+        """Return collaborators of the Master list."""
+        return {
+            CustomerId.CG_INTERNAL_CUSTOMER,
+            CustomerId.cust002,
+            CustomerId.cust003,
+            CustomerId.cust004,
+            CustomerId.cust042,
+        }
+
+
+class GenePanelCombo:
+    COMBO_1: dict[str, set[str]] = {
```

Comment:
These panel combos are general for all customers so I guess it would make more sense to call the attribute `COMBOS` and then call `GenePanelCombo.COMBOS`
```suggestion
    COMBOS: dict[str, set[str]] = {
```
But what is the advantage of having the combos in a dict as an attribute of a class if the lists will be constants anyway? Is the plan to make combos customer specific?

Reply:
Humm in that case `COMBOS` would be my best suggestion but I can understand it's not the super explanative 🤔